generators/python/fastapi/Dockerfile

```diff
@@ -6,10 +6,13 @@ ENV _TYPER_STANDARD_TRACEBACK=1
 COPY pyproject.toml ./pyproject.toml
 COPY poetry.lock ./poetry.lock
 COPY ./core_utilities/fastapi /assets/core_utilities
-COPY ./src ./src
 
 RUN pip3 install poetry &&\
   poetry config virtualenvs.create false &&\
   poetry install
 
+COPY ./src ./src
+
+RUN poetry install
```

Comment:
this was done to speed up docker builds and prevent reinstall of poetry deps again and again

Reply:
How does that do this if it was already doing 
copy -> install

Reply:
you dont have to reinstall dependencies if you change source code (thats the difference) and now we install twice